docs/cli/sessions.rst

```diff
@@ -48,6 +48,45 @@ For other options, please consult the output of ``--help``.
    * - ``--dead``
      - ``CANCELLED`` and ``TERMINATED``.
 
+Both commands offer options to specify which fields of sessions should be printed as follows.
+
+.. list-table::
+   :widths: 20 80
+   :header-rows: 1
+
+   * - Option
+     - Included Session Fields
+
+   * - (no option)
+     - ``Session ID``, ``Owner``, ``Image``, ``Type``,
+
+       ``Status``, ``Status Info``, ``Last updated``, and ``Result``.
+
+   * - ``--id-only``
+     - ``Session ID``.
+
+   * - ``--detail``
+     - ``Session ID``, ``Owner``, ``Image``, ``Type``,
+
+       ``Status``, ``Status Info``, ``Last updated``, ``Result``,
+
+       ``Tag``, ``Created At``, ``Occupied Resource``, ``Used Memory (MiB)``,
+
+       ``Max Used Memory (MiB)``, and ``CPU Using (%)``.
+
+   * - ``-f``, ``--format``
+     - ``Session ID``, ``Owner``, and specified fields by user.
```

Comment:
I think it's better to remove these two default-included fields for full customizability.
Such "templated" options are already there: `--detail`, `--id-only`.

Reply:
Just show an error sayng "at least one field is required for custom format" if there is no fields given at all.

Reply:
I changed that part. If user does not pass any argument, it shows error message saying that ``` Error: --format option requires an argument  ```

Reply:
Even though I print failure message "At least one field is required for custom format", it still prints the error message above. I think it is related to the ```click``` package.

Reply:
Ah, then it's okay to assume that there will be at least one item in the format argument.